### PR TITLE
Override 'nextSibling', 'nextElementSibling' properties (close #857)

### LINKED ISSUE #857: Override 'nextSibling', 'nextElementSibling' properties

### PR DESCRIPTION
**Pull request by churkin**
\сс @miherlosev @LavrovArtem

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/shadow-ui-test.js [hunk 1]
```diff
@@ -264,6 +271,26 @@ test('body.getElementsByClassName', function () {
     strictEqual(elems[0].id, 'pageElem');
 });
 
+test('body.getElementsByClassName', function () {
+    var root   = shadowUI.getRoot();
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/shadow-ui-test.js]: You add a copy of the test ('body.getElementsByClassName'). Remove it.

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Pull request merged**